File: LexerPLSharpVersion/LexerPLSharpVersion/constants/Separators.cs
using LexerPLSharpVersion.token;

namespace LexerPLSharpVersion.constants {
   public static class Separators {
      public const char Dot = '.';
      public const char Colon = ':';
      private const char Semicolon = ';';
      private const char Comma = ',';
      
      public static bool IsSeparator(char character) {
         switch (character){
            case Dot:
            case Comma:
            case Colon:
            case Semicolon:
               return true;
            default:
               return false;
         }
      }
      
      public static TokenType DetectSeparatorType(char character) {
         switch (character){
            case Comma:
               return TokenType.Comma;
            case Dot:
               return TokenType.Dot;
            case Semicolon:
               return TokenType.Semicolon;
            default:
               return TokenType.Unknown;
         }
      }
   }
}

File: ITEJA_Yarosh/src/main/java/lexer/constants/Separators.java
package lexer.constants;

import token.TokenType;

public class Separators {
    public static final char DOT = '.';
    public static final char COLON = ':';
    private static final char SEMICOLON = ';';
    private static final char COMMA = ',';
    public static final char QUOTE = '"';

    public static boolean isSeparator(char character) {
        switch (character){
            case DOT:
            case COMMA:
            case COLON:
            case SEMICOLON:
            case QUOTE:
                return true;
            default:
                return false;
        }
    }

    public static TokenType detectSeparatorType(char character) {
        switch (character){
            case DOT:
                return TokenType.DOT;
            case COMMA:
                return TokenType.COMMA;
            case COLON:
                return TokenType.COLON;
            case SEMICOLON:
                return TokenType.SEMICOLON;
            case QUOTE:
                return TokenType.QUOTE;
            default:
                return TokenType.UNKNOWN;
        }
    }
}
